docs/getting-started/checkout-v3.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Checkout V3
+parent: Getting Started
+nav_order: 5
+---
+
+# Checkout V3
```

Comment:
iOS has a Sequence Diagram, you can probably use the same one

Reply:
I removed it on purpose. I think is not accurate and a bit confusing.